### ISSUE TITLE
Improve mobile user experience

### ISSUE THREAD

**Issue by ForNeVeR** [codingteam/Keter#70]
Currently Keter is not accessible for mobile users who are not using the keyboard. We need to do something about it.

Are there any samples of mobile roguelike games?

**Comment by hagane** [comment]
Yes, there is a nice (and free) mobile roguelike named Dungeon of Slyn.
